.github/workflows/wix.yml

```diff
@@ -24,6 +24,7 @@ jobs:
           TAG: jdk8u362-b09
           SUB_TAG: 8u362b09
           JVM: hotspot
+          WIX_VERSION: 5.0.0
```

Comment:
guessing this isn't strictly needed if you're already setting the default to be 5.0.0? Maybe remove these and the `-wix_version ${{ env.WIX_VERSION }}` flag

Reply:
I see you point. I originally though that this could be a nice way to let developers know that this is an option (just by looking at the yaml), and that both scripts need the wix_version input. If we think it is a better option, I can just leave this info in the README.md instead

Reply:
yeah lets keep the workflow simple and document any additional configuration options in the README.md

Reply:
Done